src/M6Web/Component/RedisMock/RedisMock.php

```
@@ -795,4 +987,41 @@ protected function deleteOnTtlExpired($key)
 
         return false;
     }
+
+    protected function serializer($value)
+    {
+        return serialize($value);
+    }
+
+    protected function unserializer($value)
+    {
+        return unserialize($value);
+    }
+
+    /**
+     * @return boolean
+     */
+    public function isUseSerializer()
+    {
+        return $this->use_serializer;
+    }
+
+    /**
+     * @param boolean $serialize_set
+     */
+    public function setUseSerializer($use_serializer)
+    {
+        $this->use_serializer = $use_serializer;
+    }
+
+    protected function isSerializedValue($value)
+    {
+        $data = @unserialize($value);
```

Why need to hide errors ?